src/App.js

```diff
@@ -44,9 +44,13 @@ import ModalAlert from './components/ModalAlert';
 import SoftwareWalletWarningMessage from './components/SoftwareWalletWarningMessage';
 import AddressList from './screens/AddressList';
 
-hathorLib.network.setNetwork('mainnet');
 hathorLib.storage.setStore(new HybridStore());
 
+// Default if data on storage is null is mainnet
+// because we currently support only mainnet
+const currentNetwork = hathorLib.storage.getItem('wallet:network') || 'mainnet';
+hathorLib.network.setNetwork(currentNetwork);
```

Comment:
Why is this needed? I thought we would always perform `hathorLib.network.setNetwork` when we receive the response from the version API call.

Reply:
Yeah, I am doing it in the `src/utils/version.js` file, however this is needed for the moment I open the app and because of the way we are doing to validate the version API.

We execute async the version API call and the wallet load, so when we start loading we still don't have the version API call response. This was done because otherwise we would be 'waiting' the version API call in a blank screen. I need to do a bigger refactor to fix this and didn't want to do it in this PR

Reply:
Ok, got it. Agree on holding the refactor for later. I'd just say to put this as a comment so other people don't have this same doubt.

Reply:
Added the comment